.travis.yml

```diff
@@ -14,11 +14,13 @@ env:
     - SUITE=pg
     - SUITE=sqlite3
 rvm:
-  - 2.2.3
+  - 2.2.6
+  - 2.3.3
+  - 2.4.1
```

Comment:
Urgh, the app don't work with 2.4 yet, so maybe we can leave only 2.2 and 2.3 now?